build.gradle

```diff
@@ -31,7 +31,7 @@ buildscript {
             'junit'    : '1.1.1',
         ],
         'junit'            : '4.12',
-        'kotlin'           : '1.6.10',
+        'kotlin'           : '1.6.21',
```

Comment:
Would it be unsafe to lift Kotlin any higher than 1.6? Kotlin is at 1.9 now. OK to leave to a future PR, just thinking of taking advantage of the major version bump now.

Reply:
So we're forecasting another major version bump? Or should we make both PRs contribute to the same 4.0.0 release?

Reply:
Upgrade to 1.9 not forecasted to introduce breaking changes, ok to leave to separate PR.